testsuite/modern-testsuite/src/test/java/org/sonatype/nexus/testsuite/nuget/NugetITSupport.java

```diff
@@ -41,23 +42,35 @@
 import static org.ops4j.pax.exam.CoreOptions.wrappedBundle;
 
 /**
- * @since 3.0
+ * Support for Nuget ITs
  */
-public class NugetITSupport
+public abstract class NugetITSupport
     extends NexusHttpsITSupport
 {
+  public static final String VISUAL_STUDIO_INITIAL_COUNT_QUERY =
+      "Search()/$count?$filter=IsLatestVersion&searchTerm=''&targetFramework='net45'&includePrerelease=false";
+
+  public static final String VISUAL_STUDIO_INITIAL_FEED_QUERY =
+      "Search()?$filter=IsLatestVersion&$orderby=DownloadCount%20desc,Id&$skip=0&$top=30&searchTerm=''&targetFramework='net45'&includePrerelease=false";
+
+  public static final int VS_DEFAULT_PAGE_REQUEST_SIZE = 30;
+
+  private List<Repository> repositories = new ArrayList<>();
+
+  @Inject
+  private RepositoryManager repositoryManager;
+
   @org.ops4j.pax.exam.Configuration
   public static Option[] configureNexus() {
     return options(nexusDistribution("org.sonatype.nexus.assemblies", "nexus-base-template"),
         withHttps(),
         wrappedBundle(maven("org.apache.httpcomponents", "httpmime").versionAsInProject())
+            .overwriteManifest(OverwriteMode.FULL).instructions("DynamicImport-Package=*"),
+        wrappedBundle(maven("org.sonatype.http-testing-harness", "server-provider").versionAsInProject())
```

Comment:
I fixed the duplicate import in the metadata: https://github.com/sonatype/http-testing-harness/commit/3c1db2dabd7c399d0a54f1076baa7f52b75690f1 - for now just use wrappedBundle to rewrite the metadata as a workaround